__tests__/lib/__snapshots__/analyzeOas.test.ts.snap

```diff
@@ -144,6 +146,49 @@ exports[`#analyzeOas > should analyze an OpenAPI definition 1`] = `
         "3.1": "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#xml-object",
       },
     },
+    "xmlRequests": {
+      "description": "Any request body that accepts an XML payload.",
+      "locations": [],
+      "present": false,
+      "url": {
+        "3.0": "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#media-type-object",
+        "3.1": "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#media-type-object",
```

Comment:
thoughts on linking to this instead? these examples actually show `application/xml` payloads:

```suggestion
        "3.0": "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#request-body-object",
        "3.1": "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#request-body-object",
```